### PR TITLE
Allow installing on PHP 8.

### PR DESCRIPTION
**Pull request by ADmad**
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | N/A
| Documentation   | N/A
| License         | MIT

#### What's in this PR?

Allow installing on PHP 8.

#### Checklist

- [ ] Updated CHANGELOG.md to describe BC breaks / deprecations | new feature | bugfix
- [ ] Documentation pull request created (if not simply a bugfix)

#### To Do

- [ ] If the PR is not complete but you want to discuss the approach, list what remains to be done here

### CODE REVIEW

**Comment by ADmad** [comment]
Ping

**Review by dbu — APPROVED** [review]
thanks for the MR. PHP 8 is not yet released, i am not sure we should merge this before it is. then again its a testing library, so it would not be that much of a problem if there are any last-minute changes in php 8.

i fixed the styleci configuration error in master. your changes don't touch any php files, so it is not necessary to rebase. @Nyholm do you agree we merge this?

**Comment by ADmad** [comment]
> thanks for the MR. PHP 8 is not yet released, i am not sure we should merge this before it is. then again its a testing library, so it would not be that much of a problem if there are any last-minute changes in php 8.

People won't be able to try out the PHP 8 beta/RC release and provide feedback/bug reports if they are not able to test their code with it :slightly_smiling_face:. Testing libs like this should absolutely be updated now so as to not block the upgrading of the whole ecosystem which depend on it.

**Comment by ADmad** [comment]
Ping

**Comment by mnapoli** [comment]
🙏  please :)

**Review by sagikazarmark — APPROVED** [review]
LGTM

File: .travis.yml [hunk 1]
```diff
@@ -9,6 +9,7 @@ cache:
 php:
     - 7.2
     - 7.3
+    - 'nightly'
```
Comment (Nyholm) [inline .travis.yml:12]:
```suggestion
    - 7.4
    - 'nightly'
```

**Comment by weierophinney** [comment]
bump

RC2 just dropped, so PHP 8 is reaching relative stability. A number of PSR-7 providers are trying to ensure they are ready (e.g., Diactoros, Slim), but we are all currently blocked until this is merged.

**Comment by Nyholm** [comment]
I’ll review and make sure this is merged tonight or tomorrow. 

Thank you for the ping

**Comment by Nyholm** [comment]
Im fine this. Thank you

**Comment by Nyholm** [comment]
https://github.com/php-http/psr7-integration-tests/releases/tag/1.1.0

**Pull request merged**